css/style.css

```diff
@@ -30,146 +32,110 @@ a.ui-button:active,
-    padding: 1em 1.2em;
+	padding: 1em 1.2em;
 }
 
 .projectUsername {
-  color: black;
-}
-
-.home {
-	float:right;
-	height: 20px;
-	width:20px;
-	padding-right: 1%;
-	padding-top: 1%;
-}
-
-.panel-group {
-  margin: 10px;
-  background-color: #8fc202;
-  border-top:6px solid #8fc202;
-  border-bottom:6px solid #8fc202;
-  border-radius: 6px;
-}
-
-.panel-default>.panel-heading {
-  background-color: #8fc202;
-  border-color: #8fc202;
+	color: black;
 }
 
-.plus {
-  float:right;
-  text-shadow: 0px 0px 1px #404040;
-}
-
-.panel-default {
-  border-color: #8fc202;
-}
-
-.panel-title {
-  color: #ffffff;
-}
-
-.panel>.list-group .list-group-item, .panel>.panel-collapse>.list-group .list-group-item {
-  border-width: 0px;
-}
-
-a:hover {
-  color: #8fc202;
-  font-size: 12px;
-  cursor: pointer;
-}
+/* Inline blocks properties */
 
-a {
-  color: #8fc202;
-  font-size: 12px;
+.control {
+	background-color: #b18e35;
+	border: 1px dashed #000;
+	color: white;
+	padding:1px;
 }
 
-.panel-text {
-  color: white;
-  font-weight: bold;
-  text-shadow: 0px 0px 1px #404040;
-  font-size: 18px;
+.variables {
+	background-color: #d05f2d;
+	border: 1px dashed #000;
+	color: white;
+	padding:1px;
 }
 
-.panel-text:hover {
-  color: white;
-  font-weight: bold;
-  text-shadow: 0px 0px 1px #404040;
-  font-size: 18px;
+.procedures {
+	background-color: #7c5385;
+	border: 1px dashed #000;
+	color: white;
+	padding:1px;
 }
 
-h3 {
-  color: #555;
-  font-size: 18px;
-  font-weight: bold;
+.text {
+	background-color: #b32d5e;
+	border: 1px dashed #000;
+	color: white;
+	padding:1px;
 }
 
-body {
-  color: #555;
-  font-size: 12px;
+.getters {
+	background-color: #439970;
+	border: 1px dashed #000;
+	color: white;
+	padding:1px;
 }
 
-.btn {
-  margin: 5px;
+.setters {
+	background-color: #266643;
+	border: 1px dashed #000;
+	color: white;
+	padding:1px;
 }
 
-.tutorials {
-  background-color: #cbdca9;
-  border-width: 0px;
-  margin: 10px;
+.logic {
+	background-color: #8ab74e;
+	border: 1px dashed #000;
+	color: white;
+	padding:1px;
 }
 
-.tutorials:hover {
-  cursor: pointer;
-  background-color: #E8F2D4;
+.math {
+	background-color: #3f71b5;
+	border: 1px dashed #000;
+	color: white;
+	padding:1px;
 }
 
-.tut-nav {
-  background-color: #f3f3f3;
-  width: 150px;
+.lists {
+	background-color: #49a6d4;
+	border: 1px dashed #000;
+	color: white;
+	padding:1px;
 }
 
-.tut-navig {
-  background-color: #f3f3f3;
-  border-top:6px solid #f3f3f3;
-  border-bottom:6px solid #f3f3f3;
-  border-radius: 6px;
-}
+/* Enlarge Image feature */
 
-.tut-nav>.panel-heading{
-  background-color: #f3f3f3;
-  border-color: #f3f3f3;
+.tutorialDiv:hover .enlargeImage {
+  opacity: 0.75;
 }
 
-.tutorialNavigationHeader {
-  width: 100%;
+.left {
+	float: left;
+	max-width: 30%;
 }
 
-.tutorialContentPage {
-  background-color: #f3f3f3;
-  padding: 10px;
-  margin-right: 10px;
-  margin-left: 10px;
+.right {
+	float:right;
+	max-width: 30%;
 }
 
 /* Enlarge Image */
```

Comment:
In yr/css/style.css, the `.tutorialDiv` class also had `display: inline-block`. This version does not. I think this is the right way to do it (divs are block by default, not inline-block), which gives the appropriate vertical arrangement of sections.

Reply:
added display: inline-block;